pom.xml

```diff
@@ -138,7 +138,7 @@
 
     <dependency>
       <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-api</artifactId>
+      <artifactId>junit-jupiter</artifactId>
```

Comment:
what does this give us over API?

Reply:
It appears to be their recommendation to use this: https://github.com/junit-team/junit5-samples/blob/r5.6.0/junit5-jupiter-starter-maven/pom.xml#L20

I've found I couldn't run the unit tests in Eclipse with just using `api`, I got ClassNotFoundExceptions.